consensus/bor/bor.go

```diff
@@ -148,15 +148,15 @@ func ecrecover(header *types.Header, sigcache *lru.ARCCache) (common.Address, er
 }
 
 // SealHash returns the hash of a block prior to it being sealed.
-func SealHash(header *types.Header) (hash common.Hash) {
+func SealHash(header *types.Header, c *params.BorConfig) (hash common.Hash) {
```

Comment:
Could we do instead:
```
func (c *Bor) SealHash()...
```
and then the function signature would be the same. I think this function is not being used elsewhere.

Reply:
no
because snapshot.go doesn't have bor object